### ISSUE TITLE
[a11y] Cannot change ownership of data set

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1196]
**Describe the bug**
Cannot change ownership of WB data set

**To Reproduce**
1. Go to a WB data set
2. Tools
3. Change Owner
4. Select a user. Nothing will happen

**Screenshots**
![image](https://user-images.githubusercontent.com/37256050/158664561-8d51ccb3-c0cc-4b80-8fba-2c09122928a3.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Edge
